### ISSUE TITLE
5.1.10 launches but crashes when I open the browser

### ISSUE THREAD

**Issue by dreh65** [theotherp/nzbhydra2#887]
Using the Generic build since I'm on macOS (Monterey). Using python 3 so I use that launcher. It uses Java; I thought 5 doesn't need Java? Goes thru launching Safari but in Terminal it has now crashed. I can send the log if needed. (4.7.6 runs but usually has to run 2-3 times before it's stable and then I can usually use it after that.)

## How to create a bug report

**Set your log file level to debug, repeat the steps that cause the problem and post the debug infos ZIP which can be retrieved in the [System / Bugreport section](http://127.0.0.1:5076/system/bugreport). If other tools are involved post their logs as well.**

* Prefix the issue title with "BUG"
* You don't have to follow the next steps by the letter, just think about what you're trying to achieve ;-) 
* What do you expect to happen? What does happen? 
* Have you searched for this problem? You don't have to answer, just make sure.
* Can the problem be reproduced? How?
* Use a proper title. Describe the problem succinctly, don't just copy and paste the error message.
* If you use a docker container please tell me which one.

**I will close your issues if I you didn't provide proper information as described as above**

The more information you provide up front the higher the chance that I can help you.

## How to request a new feature
* Prefix your issue title with "REQ".
* What do you want to happen?
* If not clear, why do you want it?
* Do you think it's something only you need or something that might be popular?
* If need be explain the use case.

**Comment by theotherp** [comment]
Please do attach debug infos.

**Comment by dreh65** [comment]
[nzbhydra2.2023-08-28.log](https://github.com/theotherp/nzbhydra2/files/12500577/nzbhydra2.2023-08-28.log)

**Comment by dreh65** [comment]
sorry it took a few days. some other things came up.

**Comment by theotherp** [comment]
Nothing to see there.
Do you mean the program crashes when you open the URL in your browser? Or the website is not shown?
Please also post the file wrapper.log.

**Comment by dreh65** [comment]
Yes, NZBHydra runs all the way thru in Terminal, then when the browser
opens the program crashes.

**Comment by dreh65** [comment]
Crikey, I thought I had uploaded the wrapper.log. Was wondering why this was still in orbit. Apologies.
[wrapper.log](https://github.com/theotherp/nzbhydra2/files/12648705/wrapper.log)

**Comment by theotherp** [comment]
That's really weird. It looks like hydra is shutting down regularly but without any entry in the log why it would do so.
You said you're running it in terminal window. Do you see any error output there?

**Comment by dreh65** [comment]
This is the end of the Terminal launch:
2023-09-19 13:12:58,338  WARNING - Unable to read control ID from /Users/deh/Desktop/NZBHydra/nzbhydra2-5.1.10-generic/data/control.id: [Errno 2] No such file or directory: '/Users/deh/Desktop/NZBHydra/nzbhydra2-5.1.10-generic/data/control.id'. Falling back to process return code -5
2023-09-19 13:12:58,339  INFO - NZBHydra main process has terminated for shutdown

In the browser, a dialogue box opens with:
An unknown error occurred while communicating with NZBHydra:

{"data":null,"status":-1,"config":{"method":"POST","transformRequest":[null],"transformResponse":[null],"jsonpCallbackParam":"callback","headers":{"NzbHydra2-Handle-Errors-Generically":true,"Accept":"application/json, text/plain, */*","Content-Type":"application/json;charset=utf-8","X-XSRF-TOKEN":"62d91f3a-0ec9-4a41-a740-1e1d533bdb5c"},"url":"internalapi/search","data":{"searchRequestId":66521,"query":"Girl who played with fire  ","category":"All","mode":"search","indexers":["NZB Finder"]},"cached":false},"statusText":"","xhrStatus":"error"}

**Comment by theotherp** [comment]
Sorry, this took a while. Please put this 
[nzbhydra2wrapperPy3.zip](https://github.com/theotherp/nzbhydra2/files/12705849/nzbhydra2wrapperPy3.zip) file into `/Users/deh/Desktop/NZBHydra/nzbhydra2-5.1.10-generic` and try again. Then post the file `wrapper.log`. Before that you may check if you see any actual errors in the file.

Due to a change in the code some of the errors were previously not properly logged.

**Comment by theotherp** [comment]
Disregard that, just download the latest release and use that. It probably won't fix the crash but we should be able to see something in the log.

**Comment by dreh65** [comment]
Ok, thanks.

**Comment by dreh65** [comment]
I downloaded and started the latest version on another computer on my network that doesn't normally run nzbhydra. It ran just fine. On the computer I want it to run on 5.1.11 launches and crashes as soon as I open the window on Chrome. 4.7.6 continues to run fine for the most part. Are there items in my User Library I should remove to see if v. 5 will run without crashing?

**Comment by theotherp** [comment]
New version will report more. Please reopen if this still happens.

**Comment by dreh65** [comment]
I downloaded the 6.0 generic and ran from that folder (I'm OK with adding Downloaders and Indexers) When I ran that, it loaded just fine and then I added the Downloaders and Indexers. It started crashing and won't complete. It's the same error as originally in this thread. I also tried copying all the 6.0 files to my 4.7.6 folder, replacing all the Finder asked about. Again, it crashed. I can still run 4.7.6.
